packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -333,17 +316,52 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
             processedOptions.legend.enabled = processedOptions.series!.length > 1;
         }
 
-        this.enableConfiguredOptions(processedOptions, options);
+        /*
 
-        const themeParameters = this.getThemeParameters(activeTheme, processedOptions) as AgChartThemeParams;
-        (themeParameters as any).__palette = deepClone(activeTheme.palette);
-        (themeParameters as any).__palette.type = isObject(options.theme)
-            ? paletteType(options.theme?.palette)
-            : 'inbuilt';
-        this.resolveThemeOperations(themeParameters, themeParameters);
-        this.resolveThemeOperations(themeParameters, processedOptions);
-        this.resolveThemeOperations(themeParameters, annotationThemes);
+        TODO:
+        1 – Circular paths in `chartTheme.ts` axis default shapes
+        2 – Apply annotations like series
+        3 – Should `jsonResolveSourceWithTarget()` use `isObject()` or `isPlainObject()`? Compare tests vs `processedOptions.container` being removed by it.
+        4 – Series groupings
+        5 – Cross lines
+
+        */
 
+        /*
+
+        const { annotations, ...processedDefaults } = this.getAllThemeDefaults(activeTheme, options);
+        // const [annotationsOptions, annotationsThemes] = this.splitAnnotationsOptions(annotations);
+        // const annotationThemes = deepClone(annotationsThemes);
+
+        let processedOptions = mergeDefaults(processedOverrides, options,
+         annotationsOptions,
+          processedDefaults);
+
+        // const defaultAxes = this.getDefaultAxes(options, seriesTheme);
+        // let processedOptions = mergeDefaults(
+        //     processedOverrides,
+        //     options,
+        //     annotationsOptions,
+        //     themeDefaults,
+        //     defaultAxes
+        // );
+        // this.processAxesOptions(processedOptions, axesThemes);
+        // this.processSeriesOptions(processedOptions, activeTheme);
+
+        // this.resolveThemeOperations(themeParameters, annotationThemes);
+        
+        */
+
+        // TODO: Is this still needed? The `jsonResolveOperations()` is also a deep clone.
+        // Create isolated copy of options before we start mutations - this is performance sensitive,
+        // so we aim to only do this once in the processing flow.
```

Comment:
This was likely because of later mutations in the following method calls - it we get a fresh copy each time then it should be safe to remove.

When doing this, please make sure to verify Integrated charts use-cases for impact.